app/gui2/src/stores/graph.ts

```diff
@@ -390,7 +420,7 @@ function parseNodeExpression(offset: number, content: string, idMap: IdMap): Spa
   let spanOffset = offset
   const stack: [Span, number][] = []
 
-  const tokenRegex = /(?:(\".*?\"|[0-9]+\b)|(\s+)|([a-zA-Z0-9_]+)|(.))/g
+  const tokenRegex = /(?:(".*?"|[0-9]+\b)|(\s+)|([a-zA-Z0-9_]+)|(.))/g
```

Comment:
unnecessary `(?:)` surrounding the regex? also `\d` is exactly `[0-9]` and `\w` is exactly `[a-zA-Z0-9_]`. mostly nitpicks though

also `".*?"` could maybe be `"[^"]*"`... but also it should be easy enough to support escaping, assuming the enso language does have escaping. something like `"(?:[^\\"]|\\[\\"])*"`